pages/writing/writing.md

```diff
@@ -176,18 +179,26 @@ The available functionalities should be roughly similar to those of VSCode, at l
 You can execute a Julia script from your terminal, but in most cases that is not what you want to do.
 
 ```bash
-julia myfile.jl
+julia myfile.jl  # avoid this
 ```
 
 Julia has a rather high startup and compilation latency.
 If you only use scripts, you will pay this cost every time you run a slightly modified version of your code.
 That is why many Julia developers fire up a REPL at the beginning of the day and run all of their code there, chunk by chunk, in an interactive way.
+Full files can be run interactively from the REPL with `include`.
+
+```julia-repl
+julia> include("myfile.jl")
+```
+
+Alternatively, `includet` from the [Revise](https://timholy.github.io/Revise.jl/stable/user_reference/#Revise.includet) package can be used to "include and track" a file.
+This will automatically update function definitions in the file in the running REPL session.
 
 \vscode{
 
 [Running code](https://www.julia-vscode.org/docs/stable/userguide/runningcode/) is made much easier by the following commands:
 
-* `Julia: Start REPL` (shortcut `Alt + J` then `Alt + O`) - this is different from opening a VSCode _terminal_ and launching Julia manually from there, because the integrated Julia REPL allows you to send code from files to the REPL directly.
+* `Julia: Restart REPL` (shortcut `Alt + J` then `Alt + R`) - this is different from opening a VSCode _terminal_ and launching Julia manually from there, because the integrated Julia REPL allows you to send code from files to the REPL directly.
```

Comment:
Restart does the same thing as start if there is no REPL yet, so I always just use that, especially since the need to periodically restart the REPL is discussed immediately below these lines.

Reply:
TIL! Maybe specify that it starts AND restarts?